api/provider/aws/test_aws/service_logs_test.go

```diff
@@ -0,0 +1,207 @@
+package test_aws
```

Comment:
we may want to comment this out for now - I think it's worth having a discussion how much we want to unit test vs mock out in regards to service/task logs.

Reply:
Fair enough. ✅

Reply:
Does this code pass the test as-is?

Reply:
@diemonster Yes, this test was passing before it was commented out.